tests/integration/models/test_constitutive_laws.py

```diff
@@ -55,8 +56,6 @@
         ("momentum_balance", "friction_coefficient", 1),
         ("momentum_balance", "friction_bound", 1),
         # Energy balance
-        ("energy_balance", "bc_values_enthalpy_flux", None),
```

Comment:
Were these removed on purpose?

Reply:
Yes, bc value methods are no longer of the type DomainList -> Operator.